src/extras/simplePagination.ts

```diff
@@ -0,0 +1,76 @@
+import { stringifyVariables } from 'urql';
+import { Resolver, Variables } from '../types';
+
+export interface PaginationParams {
+  fromName?: string;
+  limitName?: string;
+}
+
+interface Base {
+  id?: any;
+  _id?: any;
+  __typename: any;
+}
+
+export const simplePagination = (params?: PaginationParams): Resolver => {
+  const from = (params && params.fromName) || 'from';
+  const limit = (params && params.limitName) || 'limit';
+
+  const compareArgs = (
+    fieldArgs: Variables,
+    connectionArgs: Variables
+  ): boolean => {
+    for (const key in connectionArgs) {
+      if (key === from || key === limit) {
```

Comment:
Now that we know what from/limit is being used by the consumer we can build our compareArgs function